docs/reference/search/profile.asciidoc

```diff
@@ -53,72 +53,90 @@ This will yield the following result:
       "failed": 0
    },
    "hits": {
-      "total": 1,
-      "max_score": 1,
+      "total": 3,
+      "max_score": 1.078072,
```

Comment:
it seems like most of the changes in this file are just changes to the timings of the parts of the queries. Is this page auto-generated or something?